test/Makefile

```diff
@@ -1,5 +1,6 @@
 CC = gcc
 CFLAGS = -Wall -Wextra -Werror -O3 $(INCLUDE)
+CFLAGS2 = -Wall -Wextra -O3 $(INCLUDE)
```

Comment:
What was the the problem with using `CFLAGS`?

Reply:
because -Werror block me

Reply:
Where?

Reply:
in my test code, the signal handler function not use parameter, and so it rises "Warning Parameter not Use" and this block compilation